client/components/NetworkList.vue

```diff
@@ -2,71 +2,185 @@
 	<div v-if="networks.length === 0" class="empty">
 		You are not connected to any networks yet.
 	</div>
-	<Draggable
-		v-else
-		:list="networks"
-		:filter="isCurrentlyInTouch"
-		:prevent-on-filter="false"
-		handle=".channel-list-item[data-type='lobby']"
-		draggable=".network"
-		ghost-class="ui-sortable-ghost"
-		drag-class="ui-sortable-dragged"
-		group="networks"
-		class="networks"
-		@change="onNetworkSort"
-		@start="onDragStart"
-		@end="onDragEnd"
-	>
-		<div
-			v-for="network in networks"
-			:id="'network-' + network.uuid"
-			:key="network.uuid"
-			:class="{
-				collapsed: network.isCollapsed,
-			}"
-			class="network"
-			role="region"
-		>
-			<NetworkLobby
-				:network="network"
-				:is-join-channel-shown="network.isJoinChannelShown"
-				@toggleJoinChannel="network.isJoinChannelShown = !network.isJoinChannelShown"
-			/>
-			<JoinChannel
-				v-if="network.isJoinChannelShown"
-				:network="network"
-				:channel="network.channels[0]"
-				@toggleJoinChannel="network.isJoinChannelShown = !network.isJoinChannelShown"
+	<div v-else ref="networklist">
+		<div class="jump-to-input">
+			<input
+				ref="searchInput"
+				:value="searchText"
+				placeholder="Jump to..."
+				type="search"
+				class="search input mousetrap"
+				aria-label="Search among the channel list"
+				tabindex="-1"
+				@input="setSearchText"
```

Comment:
Why are you using this instead of `v-model`?

Reply:
I think we needed it to get the event on each keypress due to the vue issue with composition https://github.com/vuejs/vue/issues/8723.

Reply:
Oh oops. I changed it in https://github.com/thelounge/thelounge/pull/3730 and it seemed to work. Will need to test more closely.